lab/AboutDialog/AboutDialog/BindableBase.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System.ComponentModel;
+using System.Runtime.CompilerServices;
+
+namespace Sharpen
+{
+    /// <summary>
+    ///     Implementation of <see cref="INotifyPropertyChanged" /> to simplify models.
```

Comment:
Adjust the comments to the Sharpen convention. The comment text is not indented. E.g:

```
/// <summary>
/// Implementation of <see cref="INotifyPropertyChanged" /> to simplify models.
/// </summary>
```